lessons/beginners/expressions/index.md

```diff
@@ -75,21 +75,21 @@ b = 5
 c = 3
 
 
-x = -b + (b ** 2 + 4 * a * c) ** 0.5 / (2 * a)
-#    |    |            |   |                |
-x = -5 + (5 ** 2 + 4 * 2 * 3) ** 0.5 / (2 * 2)
-#         ╰──┬─╯   ╰─┬─╯               ╰──┬──╯
-x = -5 + (  25   +   8   * 3) ** 0.5 /    4
-#                   ╰────┬─╯
-x = -5 + (  25   +      24  ) ** 0.5 /    4
-#        ╰───────┬──────────╯
-x = -5 +         49           ** 0.5 /    4
-#                ╰──────┬──────────╯
-x = -5 +               7.0           /    4
-#                      ╰─────────────┬────╯
-x = -5 +                            1.75
-#   ╰──────────────┬───────────────────╯
-x =              -3.25
+x = (-b + (b ** 2 - 4 * a * c) ** 0.5) / (2 * a)
+#     |    |            |   |                 |
+x = (-5 + (5 ** 2 - 4 * 2 * 3) ** 0.5) / (2 * 2)
+#          ╰──┬─╯   ╰─┬─╯                ╰──┬──╯
+x = (-5 + (  25   -   8   * 3) ** 0.5) /    4
+#                     ╰───┬─╯
+x = (-5 + (  25   -      24  ) ** 0.5) /    4
+#         ╰───────┬──────────╯
+x = (-5 +         1            ** 0.5) /    4
+#                 ╰──────┬──────────╯
+x = (-5 +                1)            /    4
+#    ╰────────────┬──────╯
+x =              -4                    /    4
+#                 ╰────────────┬────────────╯
+x =                           -1
```

Comment:
```suggestion
x =                           -1.0
```

Reply:
Aha, já to nespouštěla postupně v Pythonu, to mi to nedošlo. Teď jsem se to spustila postupně v Pythonu a ono už `1 ** 0.5` vrátí `1.0`, takže to bude na více místech.

Reply:
Myslím že to už není tak podstatné. Ty mezivýsledky jsou pro lidi, ne stroje.
Jen u posledního výsledku bych chtěl, aby se přesně shodoval s tím co dostaneš v Pythonu. Rozdíl by odváděl pozornost.